src/commands/build.ts

```diff
@@ -166,6 +167,19 @@ export default class Build extends BaseCommand {
         continue;
       }
 
+      for (const rule of plan.rules) {
+        if (isEmpty(rule.jsonSchema.properties)) {
+          rule.jsonSchema.properties = {
+            _: {
+              type: "string",
+              $id: "/properties/_",
+              description:
+                "This property is added as a workaround to generate type for events without properties",
+            },
+          };
+        }
```

Comment:
I hadn't actually though of this workaround! 🤣 

How about we use the JSON Schema type `null`: https://json-schema.org/understanding-json-schema/reference/null.html#null

That way people can only send `undefined` or `null` which should be harmless.

Reply:
updated